go/cmd/betabot/main.go

```diff
@@ -57,64 +63,289 @@ func betabot() error {
+						return
+					}
+					_, err = messengerClient.Interact(ctx, &bertymessenger.Interact_Request{
+						Type:                  bertymessenger.AppMessage_TypeUserMessage,
+						Payload:               userMessage,
+						ConversationPublicKey: interaction.ConversationPublicKey,
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+
+					conv.Count++
+					return
+				}
+				if conv.Count == 2 && checkValidationMessage(receivedMessage.GetBody()) {
+					// TODO invitation to real berty-community (in this group, betabot auto-reply will be disable)
+					time.Sleep(1 * time.Second)
+					_, err = messengerClient.ConversationCreate(ctx, &bertymessenger.ConversationCreate_Request{
+						DisplayName: "berty-community",
+						ContactsToInvite: []string{
+							conv.ContactPublicKey,
+						},
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+
+					time.Sleep(1 * time.Second)
+					userMessage, err := proto.Marshal(&bertymessenger.AppMessage_UserMessage{
+						Body: "OK, it’s done! Welcome here, and congrats for joining our community! 👏👍🔥\nType /help when you need infos about available test commands! 📖",
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+					_, err = messengerClient.Interact(ctx, &bertymessenger.Interact_Request{
+						Type:                  bertymessenger.AppMessage_TypeUserMessage,
+						Payload:               userMessage,
+						ConversationPublicKey: interaction.ConversationPublicKey,
+					})
+					if err != nil {
+						log.Printf("handle event: %v", err)
+						return
+					}
+
+					// delete the conv in convs
+					convs = append(convs[:idx], convs[idx+1:]...)
+					log.Printf("Finished !%v", convs)
+					return
+				}
+			}
+
+			if receivedMessage.GetBody() == "/help" {
+				userMessage, err := proto.Marshal(&bertymessenger.AppMessage_UserMessage{
+					Body: "In this conversation, you can type all theses commands :\n/demo group\n/demo demo\n/demo share\n/demo contact \"Here is the QR code of manfred, just add him!\"",
+				})
+				if err != nil {
+					log.Printf("handle event: %v", err)
+					return
+				}
+
+				_, err = messengerClient.Interact(ctx, &bertymessenger.Interact_Request{
+					Type:                  bertymessenger.AppMessage_TypeUserMessage,
+					Payload:               userMessage,
+					ConversationPublicKey: interaction.ConversationPublicKey,
+				})
+				if err != nil {
+					log.Printf("handle event: %v", err)
+					return
+				}
+				return
+			}
+
+			answers := getAnswersRange()
+			// auto-reply to user's messages
+			if err != nil {
+				log.Printf("Failed to generate randome number: %v", err)
+				return
+			}
 			userMessage, err := proto.Marshal(&bertymessenger.AppMessage_UserMessage{
-				Body: "Welcome to the beta!",
+				Body: answers[rand.Intn(len(answers))], // nolint:gosec // we need to use math/rand here, but it is seeded from crypto/rand
```

Comment:
It's really an not important random action so using an unseeded `rand` would be fine.
What gosec wants you to do is not seeding it correctly (even if that a good thing) but he wants you to use something like `math/rand#Rand` (obtained with `math/rand#New`), this isolate this random loop from others (avoiding correlation attacks) and allows for concurrent random generation (the way computers generate randomness is sequential, each thread having his own random loop allows for concurrency).